src/poliastro/twobody/orbit.py

```diff
@@ -839,6 +852,53 @@ def frozen(
                 )
             )
 
+    def calculate_pass(self, lat, long, H):
+        a = self.a  # semimajor axis
+        e = self.ecc  # eccentricity
+        i = self.inc  # equatorial inclination
+        O = self.long_asc  # longitude of ascending node
+        o = self.argp  # argument of perigee
+        T = self.period  # time of perifocal passage
+
+        n = self.n  # mean motion
+        ar = self.attractor.r  # equatorial radius
+        f = 0  # TODO: flattening
+
+        t0 = 1  # TODO: epoch universal time (rad)
+        th0 = 1  # TODO: sidereal station time (rad)
+        r_s = 1  # TODO: sidereal rate of change
```

Comment:
This should be the attractor "mean angular rotation in degrees per second". For the case of the Earth, it's not included in the 2015 WGCRE report:

> "Previous reports also included approximate expressions for the Earth. Their accuracy was poor, and the expressions failed near the fundamental epoch (J2000.0), yet they were sometimes used as a recommended model. Users should refer to the International Earth Rotation and Reference Systems Service (IERS, https://www.iers.org) for appropriate models of the Earth’s rotation."

However, we are indeed interest in a mean value, which is included here:

https://github.com/poliastro/poliastro/blob/fa131ae470b14ff1e239d6c923c4e541a8ebb0a2/src/poliastro/bodies.py#L211

And comes from the 2009 report. This is somehow coupled with https://github.com/poliastro/poliastro/issues/703.

Reply:
Wasn't this partly addressed in #735 ?

Reply:
It's unrelated - in #735 lots of values were updated, but not the ones from the Earth. In any case, what we want is to either (a) use those values, if they are suitable for a _mean_ rotational velocity, or (b) find a better value to use.

Reply:
It's unrelated - in #735 lots of values were updated, but not the ones from the Earth. In any case, what we want is to either (a) use those values, if they are suitable for a _mean_ rotational velocity, or (b) find a better value to use.

Reply:
It's unrelated - in #735 lots of values were updated, but not the ones from the Earth. In any case, what we want is to either (a) use those values, if they are suitable for a _mean_ rotational velocity, or (b) find a better value to use.

Reply:
If I recall, the only values were updated were the ones that were changed so the Earth values are up to date with the IAU 15 report

Reply:
The thing is that the Earth does not appear in IAU 15 :) but it's an unimportant detail, we can use the value from IAU 09